src/main.rs

```diff
@@ -1,40 +1,46 @@
 extern crate irc;
 
 use std::default::Default;
+use std::env;
 use irc::client::prelude::*;
 use irc::client::data::Command::PRIVMSG;
 
 fn main() {
-    let cfg = Config {
-        nickname: Some(format!("the-new-rbot")),
-        server: Some(format!("irc.freenode.org")),
-        channels: Some(vec![format!("#ayyylmao")]),
-        use_ssl: Some(true),
-        port: Some(6697),
-        .. Default::default()
-    };
-    let server = IrcServer::from_config(cfg).unwrap();
-    server.identify().unwrap();
-    for message in server.iter() {
-        // Do message processing.
-        if message.is_ok() {
-            let msg = &message.unwrap();
-            let user = msg.source_nickname().unwrap_or("");
-            if let PRIVMSG(ref chan, ref user_msg) = msg.command {
-                println!("{}: {}: {}", chan, user, user_msg);
+    let cli_args: Vec<String> = env::args().collect();
+    if cli_args.len() == 1 {
```

Comment:
Why not check if args.len() is <= 3? Maybe specify some defaults for nickname if it's not specified? What if too many arguments are provided?